src/resources/Catalogs/Catalog.ts

```diff
@@ -22,6 +22,10 @@ export default class Catalog extends Resource {
         return this.api.get<CatalogModel>(`${Catalog.baseUrl}/${catalogId}`);
     }
 
+    getFields(catalogId: string) {
+        return this.api.get<Cached<CatalogModel>>(`${Catalog.baseUrl}/${catalogId}/fields`);
```

Comment:
Wait, this should return a `CatalogFieldsModel`, not a `CatalogModel`.

Reply:
Wow, yep, totally :facepalm: